tests/generation/SettingsTest.py

```diff
@@ -27,3 +36,25 @@ def test_write_info(self):
             line = f.readline()
             
         self.assertEqual(result_comment, line)
+
+    def test_allow_complete_section_mode(self):
+        project_dir = "/repo"
+        project_files = ['/repo/hello.html']
+        ignore_globs = []
+
+        used_languages = list(get_used_languages(project_files))
+        relevant_bears = filter_relevant_bears(used_languages)
+
+        res = generate_settings(project_dir, project_files, ignore_globs,
```

Comment:
could you move the args down onto the next line, indented the same as the next line.  Makes it a bit prettier.